jorvik/storage/isolation.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+from pyspark.sql import DataFrame, SparkSession
+from pyspark.sql.streaming import StreamingQuery
+
+from jorvik.storage import Storage
```

Comment:
You cannot import from the `__init__` inside the package. The rule is the Facade see everything but no internal thing sees it.